addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11636,13 +11636,13 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#21472"
-msgid "Include All Seasons and Specials"
+msgid "Include all seasons and specials"
 msgstr ""
 
 #. Description of setting "Videos -> Library -> Include All Seasons and Specials" with label #21472
 #: system/settings/settings.xml
 msgctxt "#21473"
-msgid "Whether or not to consider All Seasons and Special items in the unwatched item selection."
+msgid "Whether or not to consider all seasons and special items in the unwatched item selection."
```

Comment:
No proper names, not proper nouns, Special is a noun but not a proper noun, only proper nouns or proper names are capitalized.

Seasons, are exempt from the uppercase summer, spring, autumn, winter except if its the forst word on a sentence.